dev/Slider/Slider_themeresources.xaml

```diff
@@ -380,7 +380,11 @@
                                     contract7Present:RadiusY="{Binding CornerRadius, RelativeSource={RelativeSource TemplatedParent}, Converter={StaticResource BottomRightCornerRadiusDoubleValueConverter}}"
                                     contract7NotPresent:RadiusX="{Binding Source={ThemeResource ControlCornerRadius}, Converter={StaticResource TopLeftCornerRadiusDoubleValueConverter}}"
                                     contract7NotPresent:RadiusY="{Binding Source={ThemeResource ControlCornerRadius}, Converter={StaticResource BottomRightCornerRadiusDoubleValueConverter}}" />
-                                <Rectangle x:Name="HorizontalDecreaseRect" Fill="{TemplateBinding Foreground}" Grid.Row="1"
+                                <!--
+                                    Using a negative margin to hide the round corners that would otherwise leave a gap between the rect and the thumb.
+                                    See https://github.com/microsoft/microsoft-ui-xaml/issues/1765 for more context.
+                                -->
+                                <Rectangle x:Name="HorizontalDecreaseRect" Fill="{TemplateBinding Foreground}" Grid.Row="1" Margin="0,0,-5,0"
```

Comment:
>5 [](start = 137, length = 1)

Does this need to scale based on the value of SliderHorizontalThumbWidth?

Reply:
Yes in theory it should. Problem is, if we use converters, it wouldn't pick up user made changes due to the fact that binding inside resource dictionaries are only evaluated once if I recall correctly.